### PR TITLE
Added ZioCarmelitano and AleCI

### PR DESCRIPTION
**Pull request by Ventus218**

### CODE REVIEW

File: src/main/java/ZioCarmelitano.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+public class ZioCarmelitano {
```
Comment (Corstor) [inline src/main/java/ZioCarmelitano.java:4]: Missing javadoc

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Ale CI
+
+on: 
+    push:
+    pull_request:
+
```
Comment (Corstor) [inline .github/workflows/ci.yml:8]: Empty new lines?

**Review by Ventus218 — COMMENTED** [review]
i think it better like it is now

File: src/main/java/ZioCarmelitano.java (hunk 1 above)
Comment (Corstor) [reply src/main/java/ZioCarmelitano.java:4]: Explain why it's named like this

**Review by Corstor — APPROVED** [review]

**Pull request merged**